src/app/modules/myinfo/myinfo.format.ts

```diff
@@ -134,3 +134,22 @@ export const formatOccupation = (
   if ('value' in field) return field.value
   return field.desc
 }
+
+/**
+ * Formats a MyInfo workpass status value
+ * by ensuring that it is in title case.
+ * Possible values are 'Live', 'Approved'.
+ */
+export const formatWorkpassStatus = (
+  field: MyInfoBasicField | undefined,
+): string => {
+  // Field value should always be a string, but check for type safety since
+  // string methods need to be called
+  if (!field || field.unavailable || typeof field.value !== 'string') return ''
+  // Change to title case
+  const originalValue = field.value
+  return (
+    originalValue.slice(0, 1).toUpperCase() +
```

Comment:
nit: can use [lodash.startCase](https://lodash.com/docs/4.17.15#startCase)

Reply:
actually that doesn't do what we want haha
<img width="509" alt="Screenshot 2021-02-24 at 5 16 12 PM" src="https://user-images.githubusercontent.com/29480346/108977897-fff6d980-76c3-11eb-98c0-e128ca6a4508.png">

Reply:
whoops: `startCase(str.toLowerCase())` would work, but yea np